package.json

```diff
@@ -64,7 +64,9 @@
     "minimist": "^1.2.0",
     "mutation-observer": "^1.0.3",
     "node-sass": "^4.8.3",
+    "patch-package": "^6.1.2",
```

Comment:
What are these new dependencies for?

Reply:
It automatically applies any patch files in the new "patches" directory at the root https://github.com/patternfly/patternfly-react/pull/2395/files#diff-1a027bc721cd1ce512f626ff77e9f5b3 - does it in a safe and predictable manner. It's basically a better alternative than using hand crafted scripts like https://github.com/patternfly/patternfly-react/pull/2395/files#diff-c5a66feeb31fe3511a6e4d157ef2bfb8 to apply changes to modules after they've landed in node_modules

Reply:
Oh right, and the whole yarn remove thing not working as expected for regular postinstall scripts https://github.com/ds300/patch-package#why-use-postinstall-postinstall-with-yarn